api/rc3.py

```diff
@@ -408,15 +408,18 @@
 	result = {} 
 	device = "/".join((domain, family, member))
 	proxy = await getDeviceProxy(device)
+	info = proxy.get_command_config(cmd)
```

Comment:
This gives a perf penalty.
Either you should cache the result or let the developer manages the correct type. They can get the type with the GET on the command.
IMHO the second option less complex regarding that tango already check the type and throw an exception.